server/database/migrations/20190530205044-create-users.js

```diff
@@ -30,7 +30,7 @@ module.exports = {
         type: Sequelize.STRING
       },
       verified: {
-        allowNull: false,
+        allowNull: true,
         type: Sequelize.BOOLEAN
```

Comment:
give this a `defaultValue` of false, `allowNull` should remain `false`

Reply:
ok